### PR TITLE
Test with Rails 5.0.0

### PR DESCRIPTION
**Pull request by bquorning**
Rails 5.0.0 is here!

### CODE REVIEW

**Comment by dasch** [comment]
I think it's fine having this here, but maybe we should wait with merging until Rails 5 is final?

**Comment by pschambacher** [comment]
:+1:

File: Gemfile [hunk 1]
```diff
@@ -7,8 +7,10 @@ platform :ruby do
...
   gem 'redcarpet'
   gem 'github-markup'
-  gem 'rails', '~> 4.2.0', require: false
+  gem 'rails', '~> 5.0.0', require: false
   gem 'rspec-rails', require: false
   gem 'benchmark-ips', require: false
   gem 'stackprof', require: false
 end
+
+gem 'genspec', github: "bquorning/genspec", ref: "rails-5-support"
```
Comment (bquorning) [inline Gemfile]: @dasch What do we do about this one?

Comment (dasch) [inline Gemfile]: I'm okay with using your branch.

**Comment by dasch** [comment]
:+1:

File: Gemfile (hunk 1 above)
Comment (bquorning) [inline Gemfile]: Until I delete it :trollface:

**Pull request merged**